neurom/fst/_neuritefunc.py

```diff
@@ -307,17 +307,40 @@ def partition_asymmetries(neurites, neurite_type=NeuriteType.all):
                              neurite_filter=is_type(neurite_type)))
 
 
-def section_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None):
-    '''Remote bifurcation angles in a collection of neurites'''
+def partition_pairs(neurites, neurite_type=NeuriteType.all):
+    '''Partition pairs at bifurcation points of a collection of neurites'''
+    return map(_bifurcationfunc.partition_pair,
+               iter_sections(neurites,
+                             iterator_type=Tree.ibifurcation_point,
+                             neurite_filter=is_type(neurite_type)))
+
+
+def section_radial_distances(neurites, neurite_type=NeuriteType.all, origin=None,
+                             iterator_type=Tree.ipreorder):
+    '''Remote bifurcation angles in a collection of neurites.
```

Comment:
I think this docstring is wrong

Reply:
I did not write this function :) Should I correct it anyways?